From 5c0ed71681bde9211beaf586e79244d7b9fb68d5 Mon Sep 17 00:00:00 2001
From: Isaku Yamahata <isaku.yamahata@intel.com>
Date: Wed, 25 May 2022 13:24:36 -0700
Subject: [PATCH 207/215] Bump KVM_EXIT_MEMORY_FAULT to avoid merge conflict

Bump KVM_EXIT_MEMORY_FAULT to avoid merge conflict with the upstream when
rebasing.

Signed-off-by: Isaku Yamahata <isaku.yamahata@intel.com>
---
 include/uapi/linux/kvm.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/uapi/linux/kvm.h b/include/uapi/linux/kvm.h
index fbc0433c8431..ea072c76dbc9 100644
--- a/include/uapi/linux/kvm.h
+++ b/include/uapi/linux/kvm.h
@@ -362,10 +362,10 @@ struct kvm_tdx_exit {
 #define KVM_EXIT_RISCV_SBI        35
 #define KVM_EXIT_RISCV_CSR        36
 #define KVM_EXIT_NOTIFY           37
-#define KVM_EXIT_MEMORY_FAULT     38
 
 /* WORKAROUND to avoid conflict with upstream. */
 #define KVM_EXIT_TDX              50
+#define KVM_EXIT_MEMORY_FAULT     100
 
 /* For KVM_EXIT_INTERNAL_ERROR */
 /* Emulate instruction failed. */
-- 
2.39.2

